.gitignore

```diff
@@ -3,6 +3,7 @@
 .settings
 .project
 .classpath
+user-interface/src/main/bundles/
```

Comment:
> It should be added to Version Control System and committed, so that other developers do not have to compile it again.

Please do not remove from version control